vault/dendron.topic.pod.v2.quickstart.md

```diff
@@ -74,16 +74,25 @@ To run a saved pod configuration, use the `Dendron: Export Pod V2`.  In the Quic
 
 ## Adding a VS Code Shortcut
 
-You can also add a command shortcut to ru your particular pod configuration.  Bring up your VS Code `keybindings.json` file by running `Preferences: Open Keyboard Shortcuts (JSON)`.  Then add the following shortcut definition: 
+You can also add a command shortcut to run your particular pod configuration. Bring up your VS Code `keybindings.json` file by running `Preferences: Open Keyboard Shortcuts (JSON)`.  Then add the following shortcut definition: 
 
-For "args", replace with your pod-ID.
+For "args", replace with {podId: <your_pod-ID>}.
 
 MacOS example, using `cmd+k t`. For Windows or Linux, this could be `ctrl+k t` instead.
 
 ```json
 {
     "key": "cmd+k t",
     "command": "dendron.exportPodv2",
-    "args": "my-pod-id"
+    "args": {"podId": "my-pod-id"}
 }
 ```
+You can also provide export scope as an argument to the command. It has higher precedence to the `exportScope` saved in the configuration file. 
```

Comment:
nit: 

"You can ... to the command. **If provided, this will override the** `exportScope` saved in the configuration file"

Reply:
Also maybe add the following sentence:

"This way, you can configure different keybindings to run the same pod configuration but with different export scopes."